scripts/data_guide/R/README.md

```diff
@@ -0,0 +1,51 @@
+# R data guide
+
+As of April 2025, we shift towards using a Python based version of the data guide instead of having to install the R dependencies.
+
+This folder contains the legacy code for the old data guide and will most likely not run as is due to pathing issues.
+
+
+## Running via Service Catalog
+Sometimes the nextflow step for the `create_data_guide.nf` will fail due to an issue with the docker image. Since troubleshooting for the docker image can easily take over a few days to resolve, try the manual approach to generate the data guide:
```

Comment:
Is there a list of known issues with the docker image to look out for along with known resolutions to those issues?

Reply:
There is a page here: https://sagebionetworks.jira.com/wiki/spaces/APGD/pages/2856943651/GENIE+Pipeline+Troubleshooting with docker related issues depending what you are working (e.g: data guide)

That doesn't yet include this latest issue which led to this ticket and PR.

Reply:
Thanks @rxu17 Perhaps that confluence page should be mentioned in this README.

If non-sage folks are reading this `README.md`, perhaps there would also be a note that it can only be accessed by Sage folks.